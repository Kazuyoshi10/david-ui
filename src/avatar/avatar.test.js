import React from 'react'
import Avatar from './avatar'
import renderer from 'react-test-renderer'

describe('Avatar', () => {
  it('should render correctly', () => {
    const component = renderer.create(
      <Avatar greeting={true}/>
    )

    expect(component.toJSON()).toMatchSnapshot()
  })
})

// test('', () => {
//   const component = renderer.create(
//     <Avatar greeting={true}/>
//   )

//   const tree = component.toJSON()
//   expect(tree).toMatchSnapshot()

// })