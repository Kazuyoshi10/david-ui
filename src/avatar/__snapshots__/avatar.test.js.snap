// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar should render correctly 1`] = `
<div
  className="avatar_f1vr1t4w"
  style={
    Object {
      "width": undefined,
    }
  }
>
  <div
    className="image_frmew96"
  >
    <img
      src="/images/david.png"
    />
  </div>
  <div
    className="message_f1xoohbz"
  >
    <div
      className="messageTail_flhpr2n"
    />
    Good evening! 
  </div>
</div>
`;
